testnet-contracts/contracts/CosmosBridge.sol

```diff
@@ -0,0 +1,131 @@
+pragma solidity ^0.5.0;
+
+import "../../node_modules/openzeppelin-solidity/contracts/math/SafeMath.sol";
+
+contract CosmosBridge {
+
+    using SafeMath for uint256;
+
+    /*
+    * @dev: Public variable declarations
+    */
+    uint256 public cosmosBridgeNonce;
+    mapping(uint256 => CosmosBridgeClaim) public cosmosBridgeClaims;
+
+    enum Status {
+        Active,
```

Comment:
can we keep this in sync with the Go implementation, ie, pending, success and failed